## [Identification of region-specific gene isoforms in the human brain using long-read transcriptome sequencing | Science Advances](https://www.science.org/doi/10.1126/sciadv.adj5279)
**Data and materials availability:** All data needed to evaluate the conclusions in the paper other than the raw sequence data and GitHub dataset are present in the paper and/or the Supplementary Materials. Iso-Seq analysis raw data have been deposited into DDBJ ([www.ddbj.nig.ac.jp/](http://www.ddbj.nig.ac.jp/)) with the BioProject accession number: PRJDB15555. The GitHub repository, containing all the code used for analysis along with the complete set of simulated datasets assumed by the code, is accessible here: [https://github.com/mihshimada/Brain_Iso-Seq](https://github.com/mihshimada/Brain_Iso-Seq) and has been archived in Zenodo with doi:[10.5281/zenodo.10398061](https://doi.org/10.5281/zenodo.10398061).


## [A spatially resolved brain region- and cell type-specific isoform atlas of the postnatal mouse brain | Nature Communications](https://www.nature.com/articles/s41467-020-20343-5)
**Data availability**
All data used for this study is publicly available on GEO under the accession token GSE158450. All data supporting the findings of this study are provided within the paper and its [supplementary information](https://www.nature.com/articles/s41467-020-20343-5#MOESM1). All additional information will be made available upon reasonable request to the authors.
**Code availability**
The source code for the methods and visualizations are available as functions in an R-package[93](https://www.nature.com/articles/s41467-020-20343-5#ref-CR93 "Joglekar, A. A spatially resolved brain region- and cell type-specific isoform atlas of the postnatal mouse brain. 
https://doi.org/10.5281/ZENODO.4278162
(2020).") ([https://github.com/noush-joglekar/scisorseqr](https://github.com/noush-joglekar/scisorseqr)) and processed single-cell and spatial data is available for download at [www.isoformAtlas.com](http://www.isoformatlas.com/)

## [Single-cell long-read sequencing-based mapping reveals specialized splicing patterns in developing and adult mouse and human brain | Nature Neuroscience](https://www.nature.com/articles/s41593-024-01616-4)
**Data availability**
The summary of all mouse data used for this study is available on the Knowledge Brain Map at [https://knowledge.brain-map.org/data/Z0GBA7V12N4J4NNSUHA/summary](https://knowledge.brain-map.org/data/Z0GBA7V12N4J4NNSUHA/summary), and all human data are available at [https://knowledge.brain-map.org/data/ASP3B09DZ8PXDUYSHDH/summary](https://knowledge.brain-map.org/data/ASP3B09DZ8PXDUYSHDH/summary). These pages contain links to raw and processed data hosted on the Neuroscience Multi-Omic data archive under the identifier dat-717krsa ([https://assets.nemoarchive.org/dat-717krsa](https://assets.nemoarchive.org/dat-717krsa)). All data supporting the findings of this study are provided within the paper and its [Supplementary Information](https://www.nature.com/articles/s41593-024-01616-4#MOESM1). Publicly available data were downloaded from APPRIS ([https://apprisws.bioinfo.cnio.es/landing_page/](https://apprisws.bioinfo.cnio.es/landing_page/)), ENCODE ([https://www.encodeproject.org/](https://www.encodeproject.org/)), GTEx ([https://www.gtexportal.org/home/downloads/adult-gtex/qtl](https://www.gtexportal.org/home/downloads/adult-gtex/qtl)) and the GWAS catalog ([https://www.ebi.ac.uk/gwas/docs/file-downloads](https://www.ebi.ac.uk/gwas/docs/file-downloads)). Source data for the main figures can be found at [https://github.com/noush-joglekar/biccn_tilgner_scisorseq/tree/main/data](https://github.com/noush-joglekar/biccn_tilgner_scisorseq/tree/main/data) (ref. [80](https://www.nature.com/articles/s41593-024-01616-4#ref-CR80 "Joglekar, A. et al. biccn_tilgner_scisorseq/data. GitHub 
https://github.com/noush-joglekar/biccn_tilgner_scisorseq/tree/main/data
(2023).")).
**Code availability**
The source code generated for this paper is publicly available at [https://github.com/noush-joglekar/biccn_tilgner_scisorseq](https://github.com/noush-joglekar/biccn_tilgner_scisorseq) (ref. [81](https://www.nature.com/articles/s41593-024-01616-4#ref-CR81 "Joglekar, A. et al. biccn_tilgner_scisorseq. GitHub 
https://github.com/noush-joglekar/biccn_tilgner_scisorseq
(2023).")).

## [Long-read transcript sequencing identifies differential isoform expression in the entorhinal cortex in a transgenic model of tau pathology | Nature Communications](https://www.nature.com/articles/s41467-024-50486-8)
**Data availability**
Raw ONT and PacBio Iso-Seq data generated in this study have been deposited in the Sequence Read Archive (SRA) database ([https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)) under accession numbers PRJNA981131 (rTg4510 targeted ONT and PacBio Iso-Seq data), PRJNA663877 (rTg4510 whole transcriptome Iso-Seq data) and PRJNA1085642 (AD post-mortem brain targeted ONT data). The processed intermediate data and UCSC genome browser tracks (merged mouse targeted data, mouse whole transcriptome Iso-Seq data, mouse sorted data and human AD targeted data) are available for download at: [https://zenodo.org/doi/10.5281/zenodo.8101907](https://zenodo.org/doi/10.5281/zenodo.8101907).
**Code availability**
All original code supporting this study is available at [https://github.com/SziKayLeung/rTg4510](https://github.com/SziKayLeung/rTg4510)[48](https://www.nature.com/articles/s41467-024-50486-8#ref-CR48 "Leung. S. Long-read transcript sequencing identifies differential isoform expression in the entorhinal cortex in a transgenic model of tau pathology. rTg4510: On Publish. 
https://doi.org/10.5281/ZENODO.12191598
(2024).") ([https://doi.org/10.5281/ZENODO.12191598](https://doi.org/10.5281/ZENODO.12191598)), and the _FICLE_ package is available to download at [https://github.com/SziKayLeung/FICLE](https://github.com/SziKayLeung/FICLE). Detailed protocols for FANS and our modified ONT library preparation approach are available on protocols.io ([https://doi.org/10.17504/protocols.io.dm6gpbwndlzp/v1](https://doi.org/10.17504/protocols.io.dm6gpbwndlzp/v1); [https://doi.org/10.17504/protocols.io.kqdg3xzwzg25/v](https://doi.org/10.17504/protocols.io.kqdg3xzwzg25/v)).


## [Long-read RNA sequencing identifies region- and sex-specific C57BL/6J mouse brain mRNA isoform expression and usage | Molecular Brain | Full Text](https://molecularbrain.biomedcentral.com/articles/10.1186/s13041-024-01112-7#data-availability)
**Data availability**
The raw dataset supporting the conclusions of this article is available in the Gene Expression Omnibus (GEO) repository, with accession number GSE246705, [https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&acc=GSE246705](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&acc=GSE246705). The data is also fully available on Sequence Read Archive (SRA) with accession number SRP469534 and BioProject with accession number PRJNA1034151.

The docker images, intermediate datasets, and code to reproduce all analyses and results in this article are available in the following Zenodo repositories: Docker images - [https://zenodo.org/records/10480924](https://zenodo.org/records/10480924), intermediate data -[https://zenodo.org/records/10381745](https://zenodo.org/records/10381745), GitHub code - [https://zenodo.org/records/10481313](https://zenodo.org/records/10481313).

The code supporting the conclusions and for reproducing analyses of this article is available in the GitHub repository, [https://github.com/lasseignelab/230227_EJ_MouseBrainIsoDiv](https://github.com/lasseignelab/230227_EJ_MouseBrainIsoDiv).

The interactive web browser application associated with this manuscript is available at [https://lasseignelab.shinyapps.io/mouse_brain_iso_div/](https://lasseignelab.shinyapps.io/mouse_brain_iso_div/).







## [Mapping medically relevant RNA isoform diversity in the aged human frontal cortex with deep long-read RNA-seq | Nature Biotechnology](https://www.nature.com/articles/s41587-024-02245-9)
**Data availability**
Raw long-read RNA-seq data generated and utilized in the present study are publicly available in Synapse[92](https://www.nature.com/articles/s41587-024-02245-9#ref-CR92 "Heberle, B. A. et al. Ebbert ebbert_lab_brain_long_read_cDNA_discovery_project. Synapse 
synapse.org/#!Synapse:syn52047893
(2023)."): [https://www.synapse.org/#!Synapse:syn52047893](https://www.synapse.org/#!Synapse:syn52047893). Raw long-read RNA-seq data generated and utilized in the present study are also publicly available in NIH Sequence Read Archive (SRA) (accession no. [SRP456327](https://www.ncbi.nlm.nih.gov/sra/?term=SRP456327))[93](https://www.nature.com/articles/s41587-024-02245-9#ref-CR93 "Heberle, B. A. et al. Ebbert ebbert_lab_brain_long_read_cDNA_discovery_project. Sequence Read Archive (SRA) 
https://trace.ncbi.nlm.nih.gov/Traces/?view=study&acc=SRP456327
(2023).") [https://trace.ncbi.nlm.nih.gov/Traces/?view=study&acc=SRP456327](https://trace.ncbi.nlm.nih.gov/Traces/?view=study&acc=SRP456327). Output from long-read RNA-seq and proteomics pipelines, reference files and annotations are publicly available at[94](https://www.nature.com/articles/s41587-024-02245-9#ref-CR94 "Heberle, B. A. et al. Ebbert Lab Nanopore PCS111 brain cDNA discovery (12 samples—AD vs controls). Zenodo 
https://doi.org/10.5281/zenodo.8180677
(2023).") [https://doi.org/10.5281/zenodo.8180677](https://doi.org/10.5281/zenodo.8180677). Long-read RNA-seq results from this article can be easily visualized through this web application: [https://ebbertlab.com/brain_rna_isoform_seq.html](https://ebbertlab.com/brain_rna_isoform_seq.html). Raw cell-line deep proteomics data utilized in this article are publicly available at [https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD024364](https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD024364). Raw brain proteomics data from round 2 of the ROSMAP TMT study are publicly available at [https://www.synapse.org/#!Synapse:syn17015098](https://www.synapse.org/#!Synapse:syn17015098). GTEx long-read RNA-seq data used for validation of our study results are available at [https://anvil.terra.bio/#workspaces/anvil-datastorage/AnVIL_GTEx_V9_hg38](https://anvil.terra.bio/#workspaces/anvil-datastorage/AnVIL_GTEx_V9_hg38). ROSMAP short-read RNA-seq data used for validation of our study results are available at [https://www.synapse.org/#!Synapse:syn21589959](https://www.synapse.org/#!Synapse:syn21589959). CHM13 reference genome sequence can be found at [https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0.fa.gz](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/analysis_set/chm13v2.0.fa.gz). CHM13 reference GFF3 annotation can be found at [https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13.draft_v2.0.gene_annotation.gff3](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13.draft_v2.0.gene_annotation.gff3). The transcript annotation from Glinos et al.[19](https://www.nature.com/articles/s41587-024-02245-9#ref-CR19 "Glinos, D. A. et al. Transcriptome variation in human tissues revealed by long-read sequencing. Nature 608, 353–359 (2022).") was retrieved from [https://storage.googleapis.com/gtex_analysis_v9/long_read_data/flair_filter_transcripts.gtf.gz](https://storage.googleapis.com/gtex_analysis_v9/long_read_data/flair_filter_transcripts.gtf.gz). The transcript annotation from Leung et al.[20](https://www.nature.com/articles/s41587-024-02245-9#ref-CR20 "Leung, S. K. et al. Full-length transcript sequencing of human and mouse cerebral cortex identifies widespread isoform diversity and alternative splicing. Cell Rep. 37, 110022 (2021).") was retrieved from [https://zenodo.org/record/7611814/preview/Cupcake_collapse.zip#tree_item12/HumanCTX.collapsed.gff](https://zenodo.org/record/7611814/preview/Cupcake_collapse.zip#tree_item12/HumanCTX.collapsed.gff).
**Code availability**
All code used in the manuscript is publicly available at [https://github.com/UK-SBCoA-EbbertLab/brain_cDNA_discovery](https://github.com/UK-SBCoA-EbbertLab/brain_cDNA_discovery) (ref. [95](https://www.nature.com/articles/s41587-024-02245-9#ref-CR95 "Heberle, B. A. et al. Brain cDNA Discovery. GitHub 
https://github.com/UK-SBCoA-EbbertLab/brain_cDNA_discovery
(2023).")).

## [Full-length transcript sequencing of human and mouse cerebral cortex identifies widespread isoform diversity and alternative splicing: Cell Reports](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721015047%3Fshowall%3Dtrue)
|                                                                              |                                                                                                                                                                                                   |                                                                                                                                                                                |
| ---------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| **Deposited data**                                                           |                                                                                                                                                                                                   |                                                                                                                                                                                |
| Raw Human Iso-Seq data                                                       | This paper                                                                                                                                                                                        | SRA: PRJNA664117                                                                                                                                                               |
| Raw Mouse Iso-Seq data                                                       | This paper                                                                                                                                                                                        | SRA: PRJNA663877                                                                                                                                                               |
| Visualization of detected full-length transcripts (Genome browser track hub) | This paper                                                                                                                                                                                        | [http://genome.ucsc.edu/cgi-bin/hgTracks?hubUrl=http://genome.exeter.ac.uk/hub/hub.txt](http://genome.ucsc.edu/cgi-bin/hgTracks?hubUrl=http://genome.exeter.ac.uk/hub/hub.txt) |
| Isoform Viewer Resource                                                      | This paper                                                                                                                                                                                        | [http://genome.exeter.ac.uk/build/index.html](http://genome.exeter.ac.uk/build/index.html)                                                                                     |
| FANTOM5 CAGE database                                                        | ([Lizio et al., 2019](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721015047%3Fshowall%3Dtrue#))  | [https://fantom.gsc.riken.jp/5/](https://fantom.gsc.riken.jp/5/)                                                                                                               |
| Intropolis junction database                                                 | [Nellore et al., 2016](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721015047%3Fshowall%3Dtrue#)  | [https://github.com/nellore/intropolis](https://github.com/nellore/intropolis)                                                                                                 |
| Human reference genome Release 38, GRCh38                                    | GENCODE                                                                                                                                                                                           | [https://www.gencodegenes.org/human/](https://www.gencodegenes.org/human/)                                                                                                     |
| Mouse reference genome Release 22, GRCm38                                    | GENCODE                                                                                                                                                                                           | [https://www.gencodegenes.org/mouse/release_M22.html](https://www.gencodegenes.org/mouse/release_M22.html)                                                                     |
| MS-based human adult and fetal proteomic dataset                             | [Kim et al., 2014](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721015047%3Fshowall%3Dtrue#)      | PXD000561                                                                                                                                                                      |
| RNA-Seq Mouse                                                                | [Castanho et al., 2020](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01504-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721015047%3Fshowall%3Dtrue#) | N/A                                                                                                                                                                            |
| RNA-Seq Fetal                                                                | This Paper                                                                                                                                                                                        | N/A                                                                                                                                                                            |
| PolyA motif list                                                             | Elizabeth Tseng                                                                                                                                                                                   | [https://github.com/Magdoll/SQANTI2](https://github.com/Magdoll/SQANTI2)                                                                                                       |

## [Full-length RNA transcript sequencing traces brain isoform diversity in house mouse natural populations](https://genome.cshlp.org/content/early/2024/10/17/gr.279166.124)
[Full-length RNA transcript sequencing traces brain isoform diversity in house mouse natural populations - Edmond](https://edmond.mpg.de/dataset.xhtml?persistentId=doi:10.17617/3.3RLIHR)

## [Long read sequencing reveals novel isoforms and insights into splicing regulation during cell state changes | BMC Genomics | Full Text](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08261-2#change-history)
The datasets supporting the conclusions of this article are available in the following repositories: read data are archived in ENA (short reads: PRJEB44501, long reads: PRJEB44502). Analysis scripts are available from [https://github.com/TGAC/SHSY5Y_differentiation](https://github.com/TGAC/SHSY5Y_differentiation) and the accompanying datafiles from [https://doi.org/10.5281/zenodo.4727856](https://doi.org/10.5281/zenodo.4727856) or the corresponding author.